.mrconfig

```diff
@@ -32,24 +32,13 @@ checkout =
 checkout =
     git clone git@github.com:ovro-lwa/lwa-issues.git
 
-[dada2ms]
-checkout =
-    git clone git@github.com:ovro-lwa/dada2ms.git
-
 [caltech-lwa]
 checkout =
     git clone https://github.com/realtimeradio/caltech-lwa
     cd caltech-lwa
     git submodule init control_sw/libs/casperfpga
     git submodule update
 
-[caltech-bifrost-dsp]
-checkout = 
-    git clone https://github.com/realtimeradio/caltech-bifrost-dsp.git
-    cd caltech-bifrost-dsp
-    git submodule init
-    git submodule update
-
-[bifrost]
```

Comment:
what happened to bifrost?

Reply:
I'd like to work our way up to the more complex builds. First step is a working deployment on cal-im with basic python.

Reply:
it's ok to pull the repo, if it doesn't have a deploy script then the service won't run it.